src/composables/theme.ts

```diff
@@ -33,8 +33,7 @@ export function useRotkiTheme(): ThemeContent {
    */
   const isLight: ComputedRef<boolean> = computed(
     () =>
-      (get(isAutoControlled) && get(system) === ThemeMode.light)
-      || get(state) === ThemeMode.light,
+      get(state) === ThemeMode.light,
```

Comment:
nitpick: can we do this and the next to single line? we should have enough space